diff -ruN poppler-0.63.0/goo/gfile.cc poppler-0.63.0-patched/goo/gfile.cc
--- poppler-0.63.0/goo/gfile.cc	2018-03-19 02:23:49.000000000 +0800
+++ poppler-0.63.0-patched/goo/gfile.cc	2018-03-19 11:34:54.536020000 +0800
@@ -80,7 +80,7 @@
 struct StatMtim
 {
   static const struct timespec& value(const Stat& stbuf) {
-    return stbuf.st_mtim;
+    return *(new timespec{stbuf.st_mtime, 0});
   }
 };
 
